release notes/v1.0.0.md

```diff
@@ -0,0 +1,106 @@
+# Grafana k6 v1.0 is here! 🎉
+
+After 9 years of iteration and countless community contributions, we're thrilled to announce **Grafana k6 v1.0**. 
+
+While many the features and capabilities in this release were introduced gradually over past versions, **k6 v1.0 marks a turning point**: a commitment to stability, formal support guarantees, and transparency in how we evolve and develop the project from here. This milestone is more than a version number; it's about trust, reliability, and empowering you to test with confidence.
+
+## Thank You, k6 Community! 🌍
+
+This release wouldn't be possible without you:
+
+- ⭐️ 27000+ GitHub stars
+- 🧠 9000+ git commits
+- 🤝 200+ contributors
+- 🔁 Countless test runs of any scale, in every timezone.
+
+
+It's been amazing to watch k6 grow, from a simple load testing CLI into a comprehensive reliability tool, used by teams worldwide and supported by a passionate and dedicated community.
+
+To everyone who filed bugs, built extensions and libraries, or championed k6:️
+**Thank you!** You've shaped what k6 is today 🙇‍♂️ 
+
+## What's New in k6 1.0?
+
+### 1. Stability You Can Build On
+
+- ✅ **Semantic Versioning**: k6 now follows [semantic versioning 2.0](https://semver.org/). Breaking changes will only happen in major releases, with prior deprecation warnings.
+- 🔒 **2-Year Support Guarantees**: Every major version will be supported with critical fixes for **at least two years**; upgrade on your schedule.
+- 📦 **Public API Surface**: We've established a clearly delineated and supported API surface for the k6 codebase. Extensions, integrations, and projects building on top of the k6 code now have a stable foundation to rely on.
+
+🔎 Read more in our [Versioning & Stability guarantees](https://grafana.com/docs/k6/latest/reference/versioning-and-stability-guarantees/) guide.
```

Comment:
```suggestion
🔎 Read more in our [versioning and stability guarantees](https://grafana.com/docs/k6/latest/reference/versioning-and-stability-guarantees/) guide.
```

Reply:
I usually try to match the page title, so I'd use:

```suggestion
🔎 Read more in our [Versioning and stability guarantees](https://grafana.com/docs/k6/latest/reference/versioning-and-stability-guarantees/) guide.
```